src/lib/helpers/config.helper.ts

```diff
@@ -26,6 +26,9 @@ export const ADDRESS_PERMISSIONS_ARRAY_KEY =
 
 export const PREFIX_PERMISSIONS = '0x4b80742d0000000082ac0000';
 export const ALL_PERMISSIONS = '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
+// exclude DELEGATECALL permission for safety
+export const DEFAULT_PERMISSIONS =
+  '0x00000000000000000000000000000000000000000000000000000000000003bf';
```

Comment:
Can we use `ERC725.encodePermission` for this? So we can move away from hardcoding permissions values